src/main/groovy/com/marklogic/gradle/task/roxy/RoxyGenerateTestScaffoldTask.groovy

```diff
@@ -0,0 +1,100 @@
+package com.marklogic.gradle.task.roxy
+
+import org.gradle.api.tasks.TaskAction
+
+//import com.marklogic.gradle.task.roxy.scaffoldTemplates.SampleTestFiles
+
+// Task mlGenerateRoxyTest : This task uses Groovy's SimpleTemplateEngine to customize the sample test suite
+// Parameters:
+//    roxySuitesDir   (Default: roxy/src/test/suites)
+//        - Specify the target directory for the sample test suite
+//    suiteName       (Default: SampleTestSuite)
+//        - Specify the name of the generated test suite
+//    testName        (Default: SampleTest)
+//        - Specify the name of the generated test
+// Examples:
+//    gradle mlGenerateRoxyTest
+//    gradle mlGenerateRoxyTest -PsuiteName="foo" -PtestName="bar" -ProxySuitesDir="./src/roxyTest/suites"
+class RoxyGenerateTestScaffoldTask extends RoxyTask {
```

Comment:
I'm not sure what the naming should be here. The task above is "GenerateTest", but it generates more than a test - it generates a test module, a setup module, a suite setup, and a suite teardown. So it seems it should be "GenerateTestSuite", and thus "mlRoxyGenerateTestSuite". To match the task name, the class name should then be "RoxyGenerateTestSuiteTask", and the use of "scaffold" below should change to "suite".